### DISCUSSION TITLE
New warning in Rider I am getting in preview7 about IAssetLoader.Open

### DISCUSSION THREAD

**Discussion by aldelaro5** [AvaloniaUI/Avalonia#11150] [Q&A] [answered]
I tested upgrading to preview7 and I got this error Rider reports (not actual build error, but still worries me):
```
CS0618: Method 'Avalonia.Platform.IAssetLoader.Open(Uri, Uri?)' is obsolete: This is a part of unstable API and can be changed in minor releases. You have been warned
```
The problem I have with this is that it doesn't tell me what I should be using. I don't understand why this warning is useful here: if it's going to go away, then what should I do? In my case, I want to load static csv files from avares which this has been working quite well and from what I understand, how you want to do it if you want to target desktop as well as wasm, but then why is this warning worded like this if I didn't really see how I was supposed to do this?

**Comment by kekekeks** [comment, marked as answer]
We've enabled this warning for everything that was previously marked as `[Unstable]` in case of asset loader, we'll introduce a proper stable API before the next preview/rc. For now you can ignore that warning.

#11155
